common/app/conf/switches/DiscussionSwitches.scala

```diff
@@ -43,4 +43,14 @@ trait DiscussionSwitches {
     exposeClientSide = true
   )
 
+  val DiscussionStopTrafficFromDotcom = Switch(
```

Comment:
Yeah I agree actually. It will stop the double negatives in the javascript as well.